File: JAVA/ex00/Multiples_of_three_five.java
public	class Multiples_of_three_five
{
	public static void main(String []args)
	{
		System.out.println(multiples_of_3_5(1000));
	}

	public	 static int	multiples_of_3_5(int nbr)
	{
		int	i;
		int	sum;

		sum = 0;
		i = 0;
		while (i < nbr)
		{
			if ((i%3 == 0) || (i%5 == 0))
				sum += i;
			i++;
		}
		return sum;
	}
}

File: C/ex00/multiples_of_three_five.c
int	multiple_of_three_five(int nbr)
{
	int i;
	int sum;

	i = 0;
	sum = 0;
	while (i < nbr)
	{
		if ((i % 3 == 0) || (i % 5 == 0))
			sum += i;
		i++;
	}
	return sum;	
}

#include <stdio.h>
int	main(int ac, char **av)
{
	printf("%d", multiple_of_three_five(49));
}
